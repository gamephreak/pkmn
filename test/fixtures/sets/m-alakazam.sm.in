Alakazam-Mega @ Alakazite  
Ability: Magic Guard  
EVs: 252 SpA / 4 SpD / 252 Spe  
Timid Nature  
IVs: 0 Atk  
- Psychic  
- Focus Blast  
- Shadow Ball  
- Recover  

