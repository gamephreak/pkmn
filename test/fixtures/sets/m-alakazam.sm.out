Alakazam-Mega (F) @ Alakazite  
Ability: Magic Guard  
EVs: 252 SpA / 252 Spe  
Timid Nature  
IVs: 0 Atk  
- Psychic  
- Focus Blast  
- Shadow Ball  
- Recover  

